packages/nl-design-system-unstable/__docs__/form-patterns.md

```diff
@@ -1,139 +1,270 @@
-# Forms
+# Richtlijnen voor formulieren
 
-For individual form component guidelines please consult the documentation for that specific component.
+_Status: **In ontwikkeling**_
 
-_Status: **In development**_
+_Individuele formulier elementen en hun specifieke richtlijnen zijn ondergebracht in de componentenbibliotheek._
 
-## Introduction
+Met een formulier nodig je iemand uit tot een dialoog met jouw organisatie. Het is niet slechts een manier om jouw database te vullen maar een onderdeel van de gehele reis die iemand door jouw product maakt. Ontwerp een formulier vanuit degene die deze in moet vullen.
```

Comment:
Ik zou deze paragraaf en degene hieronder denk ik omdraaien. De laatste zin "Ontwerp een formulier..." loopt denk ik nog lekkerder als het een mini conclusie/call to action is: "Ontwerp een formulier **dus**..."

Reply:
Ik neem deze openingsalinea’s opnieuw onder de loep.